src/lib/interfaces/digital-asset-deployment.ts

```diff
@@ -28,11 +28,24 @@ export interface DeployedLSP7DigitalAsset {
   LSP7DigitalAsset: DeployedContract;
 }
 
-export interface ContractDeploymentOptions {
+export interface ContractDeploymentOptionsReactive {
   version?: string;
   byteCode?: string;
   libAddress?: string;
-  deployReactive?: boolean;
+  deployReactive: true;
   deployProxy?: boolean;
   uploadOptions?: UploadOptions;
 }
+
+export interface ContractDeploymentOptionsNonReactive {
+  version?: string;
+  byteCode?: string;
+  libAddress?: string;
+  deployReactive?: false;
+  deployProxy?: boolean;
+  uploadOptions?: UploadOptions;
+}
```

Comment:
To avoid duplication and to make the difference between the 2 interfaces, I suggest:

```ts
interface ContractDeploymentOptionsBase {
  version?: string;
  byteCode?: string;
  libAddress?: string;
  deployReactive?: boolean;
  deployProxy?: boolean;
  uploadOptions?: UploadOptions;
}

export interface ContractDeploymentOptionsReactive extends ContractDeploymentOptionsBase {
  deployReactive: true;
}

export interface ContractDeploymentOptionsNonReactive extends ContractDeploymentOptionsBase {
  deployReactive?: false;
}
```